### PR TITLE
add logo and navbar

### PR DESCRIPTION
**Pull request by Mona904**

### CODE REVIEW

File: components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+  const handleTabClick = (tabName) => {
+    setSelectedTab(tabName);
+  };
+
+  return (
+    <FooterContainer>
+      <TabBarItem
+        className={selectedTab === "home" ? "selected" : ""}
+        onClick={() => handleTabClick("home")}
```
Comment (HenrikeStahlhut) [inline components/Footer.js:48]: Do you really need this? Because when a page is selected and the footer is visible, nothing changes?

**Review by doctype-Melvin — COMMENTED** [review]
Hi Mona,
nice job! That's some nice clean code!

**Review by HenrikeStahlhut — COMMENTED** [review]
Just one little improvement :)

File: components/Footer.js (hunk 1 above)
Comment (Mona904) [reply components/Footer.js:48]: No, I had previously inverted the colors when the tab was pressed but it looked strange, will remove, thank you!

**Pull request merged**